main.go

```diff
@@ -56,8 +56,8 @@ func CreateHandler(conf Config, git Git, pullRequests PullRequests, repositories
 	}
 }
 
-// startsWithPlusOne matches strings that start with either a +1 (not followed by other digits) or a :+1: emoji
-var startsWithPlusOne = regexp.MustCompile(`^(:\+1:|\+1($|\D))`)
+// isPlusOneComment matches strings that contain either a +1 (not followed by other digits) or a :+1: emoji
+var isPlusOneComment = regexp.MustCompile(`(:\+1:|\+1($|\D))`)
```

Comment:
I don't think you need `:\+1\:` because it is already matched with the second matcher. `(\+1($|\D))` should be enough.